### PR TITLE
OCP4 configurations and settings

### PR DESCRIPTION
**Pull request by WadeBarnes**
The OCP4 migration is complete. There are just a few items that need to be tidied up before we merge this branch.

Primarily clean up the build refs so all of the builds and pipelines are pointing at the appropriate main repository and branch.

```
agent               https://github.com/bcgov/aries-vcr.git               master
api                 https://github.com/bcgov/aries-vcr.git               master
backup              https://github.com/BCDevOps/backup-container.git     2.3.2
db                  https://github.com/wadeking98/aries-vcr.git          master
frontend            -                                                    -
frontend-artifacts  https://github.com/bcgov/aries-vcr.git               master
frontend-pipeline   https://github.com/bcgov/orgbook-configurations.git  master
frontend-runtime    https://github.com/bcgov/aries-vcr.git               master
pvc-migrator        https://github.com/BCDevOps/StorageMigration.git     master
schema-spy          https://github.com/bcgov/SchemaSpy.git               master
search-engine       https://github.com/wadeking98/aries-vcr.git          master
search-engine-base  https://github.com/bcgov/openshift-solr.git          solr-8.8.1
```

### CODE REVIEW

**Comment by WadeBarnes** [comment]
Build refs are all cleaned up now:
```
agent               https://github.com/bcgov/aries-vcr.git               master
api                 https://github.com/bcgov/aries-vcr.git               master
backup              https://github.com/BCDevOps/backup-container.git     2.3.2
db                  https://github.com/bcgov/aries-vcr.git               master
frontend            -                                                    -
frontend-artifacts  https://github.com/bcgov/aries-vcr.git               master
frontend-pipeline   https://github.com/bcgov/orgbook-configurations.git  master
frontend-runtime    https://github.com/bcgov/aries-vcr.git               master
schema-spy          https://github.com/bcgov/SchemaSpy.git               master
search-engine       https://github.com/bcgov/aries-vcr.git               master
search-engine-base  https://github.com/bcgov/openshift-solr.git          8.8.1
```

File: openshift/templates/frontend/frontend-deploy.bc.dev.param [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
+# NAMESPACE=8ad0ea
+# SUFFIX=-bc
+# APP_NAME=orgbook
+# APP_GROUP=aries-vcr
+# IMAGE_NAMESPACE=8ad0ea-tools
 # REAL_IP_FROM=172.51.0.0/16
-# API_URL=http://api-indy-cat:8080/api
-# API_SERVICE_NAME=api-indy-cat
+# API_URL=http://api-bc:8080/api
+# API_SERVICE_NAME=api-bc
```
[the remark this answers is not in the document]
Comment (WadeBarnes) [reply openshift/templates/frontend/frontend-deploy.bc.dev.param:16]: Agreed, separate issue.

File: openshift/templates/api/api-deploy.bc.dev.param [hunk 2]
```diff
@@ -35,7 +41,7 @@ TAG_NAME=dev
 # USER_PASSWORD=[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]{16}
 # ADMIN_USER_ID=[a-zA-Z_][a-zA-Z0-9_]{10}
 # ADMIN_PASSWORD=[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]{16}
-APPLICATION_URL=https://dev.orgbook.gov.bc.ca
+APPLICATION_URL=https://orgbook-dev.apps.silver.devops.gov.bc.ca
```
[the remark this answers is not in the document]
Comment (WadeBarnes) [reply openshift/templates/api/api-deploy.bc.dev.param]: Addressed, using `gov.bc.ca` URL.

File: openshift/templates/api/api-deploy.bc.prod.param [hunk 3]
```diff
@@ -35,7 +41,7 @@ TAG_NAME=bc-prod
 # USER_PASSWORD=[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]{16}
 # ADMIN_USER_ID=[a-zA-Z_][a-zA-Z0-9_]{10}
 # ADMIN_PASSWORD=[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]{16}
-APPLICATION_URL=https://orgbook-prod.pathfinder.gov.bc.ca
+APPLICATION_URL=https://orgbook-prod.apps.silver.devops.gov.bc.ca
```
[the remark this answers is not in the document]
Comment (WadeBarnes) [reply openshift/templates/api/api-deploy.bc.prod.param]: Addressed, using `gov.bc.ca` URL.

**Review by esune — APPROVED** [review]
Looks good! 👍🏻

**Pull request merged**